tests/unit/s2n_self_talk_session_resumption_test.c

```diff
@@ -32,6 +39,7 @@ static int s2n_test_session_ticket_cb(struct s2n_connection *conn, void *ctx, st
     EXPECT_SUCCESS(s2n_session_ticket_get_data_len(ticket, &data_len));
 
     struct s2n_stuffer *stuffer = (struct s2n_stuffer *) ctx;
+    EXPECT_SUCCESS(s2n_stuffer_wipe(stuffer));
```

Comment:
I *think* adding this means that you can remove all the stuffer_rewrite calls in this file.

Reply:
I probably could, but they don't hurt so I wanted to avoid all those changes. There are a lot of stuffer_rewrite calls xD